cellarium/ml/transforms/z_score.py

```diff
@@ -41,8 +41,10 @@ def __init__(
         eps: float = 1e-6,
     ) -> None:
         super().__init__()
-        self.mean_g = mean_g
-        self.std_g = std_g
+        self.mean_g: torch.Tensor
+        self.std_g: torch.Tensor
+        self.register_buffer("mean_g", mean_g)
```

Comment:
Can you convert `mean_g` and `std_g` to torch.Tensor with `torch.as_tensor(mean_g)` and  `torch.as_tensor(std_g)`? `register_buffer` expects `torch.Tensor` but `mean_g` and `std_g` can also be floats.

Reply:
Got it, thanks! Much better than my idea of checking for float and using `torch.Tensor([mean_g])`